src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,46 @@
-export const PeopleFilters = () => {
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+export const allCenturies = ['16', '17', '18', '19', '20'];
```

Comment:
it's a constant variable, so it's name should be uppercased
```suggestion
export const ALL_CENTURIES= ['16', '17', '18', '19', '20'];
```